style.css

```diff
@@ -42,3 +42,11 @@ div.inner-content {
 	background-color: green;
 	width: 100%;
 }
+
+.tabs {
```

Comment:
Watch out - the CSS class here does not match up with the one you wrote in your HTML.